ci/rpm_build_success.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Script to be run on successful RPM build
+
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ci_envs="$mydir/ci/parse_ci_envs.sh"
+if [ -e "${ci_envs}" ]; then
+  # shellcheck disable=SC1091
+  source "${ci_envs}"
+fi
+
+: "${CHROOT_NAME:='epel-7-x86_64'}"
+: "${TARGET:='centos7'}"
+
+mockroot="/var/lib/mock/${CHROOT_NAME}"
+(cd "$mockroot/result/" && cp -r . "$OLDPWD/artifacts/${TARGET}"/)
+createrepo "artifacts/${TARGET}/"
+rpm --qf %{version}-%{release}.%{arch} \
```

Comment:
(lint) In POSIX sh, brace expansion is not supported. [SC2039]
(lint) In POSIX sh, brace expansion is not supported. [SC2039]
(lint) In POSIX sh, brace expansion is not supported. [SC2039]